src/test/java/com/lob/client/test/AddressTest.java

```diff
@@ -27,6 +27,9 @@
     @Test
     public void testListAddresses() throws Exception {
         final AddressResponseList addresses = client.getAddresses().get();
+        assertEquals(200, addresses.getStatusCode());
```

Comment:
iiiinteresting. in that case, there are some that are `assertEquals(addresses.getStatusCode(), 200);`, so you should probably change them. lol